exercise_3/src/Teacher.cpp

```diff
@@ -0,0 +1,25 @@
+#include <stdexcept>
+#include <sstream>
+#include <string>
+#include "Teacher.h"
+
+Teacher::Teacher(std::istringstream &record) {
+    std::string buf;
```

Comment:
Same comment as for constructor for class Course.

Reply:
Done